docs/reference/transform/limitations.asciidoc

```diff
@@ -275,3 +275,13 @@ accessed. If the time field is a {ref}/runtime.html[runtime field], the
 performance impact of calculating field values at query time can significantly 
 slow the {transform}. Use an indexed field as a time field when using 
 {transforms}.
+
+
+[discrete]
+[[transforms-ccs-limitation]]
+=== {transforms-cap} supports {ccs} if the remote cluster configured properly
+
+If you use <<modules-cross-cluster-search,{ccs}>>, the remote cluster must 
+support the search and aggregations you use in your {transforms}. 
+{transforms-cap} validate their configuration, if it fails and you use {ccs}, 
```

Comment:
Not mandatory, but I wasn't sure if "it" referred to the transform or the validation or the configuration:
```suggestion
{transforms-cap} validate their configuration; if you use {ccs} and the validation fails, 
```